bin/bump-version

```diff
@@ -92,6 +93,17 @@ update_lock_file() {
   bundle install
 }
 
+update_example_lock_file() {
+  echo ""
+  echo "🔓 Updating example lock file"
+  pushd ./examples/split-by-agency/
+  bundle install
+
+  echo "🧪 Test the example works with the new version"
+  bundle exec split_by_agency.rb -i ../../spec/fixtures/sample_gtfs.zip --ids DTA
+  popd
```

Comment:
If we're already making a release this will likely pass, but I figured I should include this extra step as a smoke test just in case.